test-requirements.txt

```diff
@@ -78,21 +78,21 @@ drf-jwt==1.19.2
     # via
     #   -r requirements/base.txt
     #   edx-drf-extensions
-edx-django-utils==4.4.2
+edx-django-utils==4.5.0
     # via
     #   -r requirements/base.txt
     #   edx-drf-extensions
 edx-drf-extensions==8.0.1
     # via -r requirements/base.txt
 edx-lint==5.2.1
     # via -r requirements/test.in
-edx-opaque-keys==2.2.2
+edx-opaque-keys==2.3.0
     # via
     #   -r requirements/base.txt
     #   edx-drf-extensions
 factory-boy==3.2.1
     # via -r requirements/test.in
-faker==11.3.0
+faker==12.0.0
```

Comment:
Major version bump but it isn't being directly used in the tests so it should be okay to upgrade.